Ginger/GingerCoreCommon/Run/RunSetActions/RunSetActionHTMLReportSendEmail.cs

```diff
@@ -125,6 +145,10 @@ public string MailUser
             }
         }
 
+        public RunSetActionHTMLReportSendEmail()
```

Comment:
Its a new method and its not called from anywhere in the current changes. Do we need this method?

Reply:
this constructor sets the default value for Email property. We don't set the property directly to the mEmail field because in that case it won't start dirty tracking.